spec/controllers/authentication_controller_controller_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe AuthenticationControllerController, type: :controller do
```

Comment:
Unless your writing specs, I would remove these files. Kinda sends a false sense of security that you have tests files but nothing is in them.